lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -191,7 +183,7 @@ sub streaming {
 
     # setup a function to stop streaming again
     my $timer_id;
-    my $close_connection = sub {
+    my $close_connection = sub ($self, @) {
```

Comment:
What does the `@` do here?

Reply:
It consumes [0;:infinity:[ arguments into an unnamed parameter and ignores them.